packages/@magic-sdk/react-native-expo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@magic-sdk/react-native-expo",
-  "version": "19.4.1",
+  "version": "19.4.2",
```

Comment:
Do we still need this version bump?

Reply:
Yes, otherwise the publishing job will fail to update NPM with the correct version.